lib/PaginationEmbed.js

```diff
@@ -16,6 +16,7 @@ class PaginationEmbed {
      * @property {Boolean} options.showPageNumbers Whether or not to show the current page index over the embed. Defaults to: true
      * @property {Boolean} options.extendedButtons Whether or not to show extended control buttons besides standard pagination (First & Last page, deleting)
      * @property {Number} options.maxMatches How often the reaction handler should listen for a reaction (How often the paginator can be used). Defaults to: 50. Maximum: 100
+     * @property {Boolean} options.cycling If back to rist page when you click on next in last page or click last page when you click on back in first page. Defaults to: false

```

Comment:
The english isn't in line with the rest of the documentation and contains typos.

```suggestion
     * @property {Boolean} options.cycling Cycle through all embeds jumping from the first page to the last page on going back and from the last page to the first page going forth. Defaults to: false
```